dex/msgjson/types.go

```diff
@@ -55,6 +55,7 @@ const (
 	RPCPreRegisterError               // 39
 	RPCRegisterError                  // 40
 	RPCArgumentsError                 // 41
+	RPCLoginError                     // 42
```

Comment:
After `RPCInitError` to match ordering in other areas?

Reply:
Think it's necessary now? I definitely thing it's good to be organized, but as nothing is really finalized, it seems like a waste of time atm. Can this be done later? I'll make an issue.